app/components/Footer/index.js

```diff
@@ -16,80 +17,105 @@ type Props = {
 };
 
 const Footer = ({ loggedIn }: Props) => (
-  <footer className={styles.footer}>
-    <div className={styles.footerContent}>
-      <Flex column>
-        <Flex>
-          <div className={cx(styles.section, utilityStyles.hiddenOnMobile)}>
+  <div>
```

Comment:
The whole `<footer>` is wrapped in a `<div>`, has this any `css` effect, or can it be removed?

Reply:
Nice catch, it can definitely be removed! 😍 I believe this was left from back when I was experimenting and trying to figure out how css `composes` actually works ...